website/content/en/docs/advanced-topics/heterogeneous.md

```diff
@@ -0,0 +1,400 @@
+---
+title: Heterogeneous Cluster Support
+linkTitle: Heterogeneous Cluster Support
+weight: 80
+---
+## Overview
+A heterogeneous cluster is a cluster in which the nodes differ from each other in a way that makes their containers incompatible with each other. The most common incompatibility that applies is CPU architecture. The rest of this document assumes that the user is developing an application that needs to be able to deploy cleanly in a cluster with a mix of node CPU architectures. Because an operator deployed to a heterogeneous cluster may not support all of the architectures in the cluster, it is important to set up safeguards that prevent scheduling pods on nodes of incompatible architectures. These scheduling implications affect both the operator as well as its operands.
+
+### Determining the Supported Architectures for an Operator/Operand Image
+For a given container image, you can check which platforms are supported by listing them by inspecting the manifest.
+```bash
+docker manifest inspect <image>
+```
+If the above command fails, it may be because the referenced name points to a single image rather than a manifest list. In this case, you can find the architecture by inspecting the image.
+```bash
+docker inspect <image>
+```
+
+Here's an example of the architectures distributed for the Alpine Linux container image.
```

Comment:
WDYT about add the commend as example which contains the image and the whole result with arch and os which lead a better understanding

Reply:
Do you mean the full output for both commands? (I.e. docker inspect & docker manifest inspect)

I felt like the output for Alpine was too long, which is why I truncated it via the pipe to grep. I can provide a truncated version of the full output if that's preferable.

Reply:
I agree with @jaypoulz, I'd much rather see the limited list of items that are relevant than a bunch of noise. I can run the command myself to see that.

Reply:
@jaypoulz see: https://github.com/operator-framework/operator-sdk/pull/5983/files#r967708843
You do not need to add all just copy and paste 2 so and archs. 

Example:

```json
...
{
         "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
         "size": 528,
         "digest": "sha256:fa30af02cc8c339dd7ffecb0703cd4a3db175e56875c413464c5ba46821253a8",
         "platform": {
            "architecture": "ppc64le",
            "os": "linux"
         }
      },
      {
         "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
         "size": 528,
         "digest": "sha256:c2046a6c3d2db4f75bfb8062607cc8ff47896f2d683b7f18fe6b6cf368af3c60",
         "platform": {
            "architecture": "s390x",
            "os": "linux"
         }
      }


...
```